src/electron/office/office_client.cc

```diff
@@ -176,10 +177,13 @@ std::string OfficeClient::GetLastError() {
 
 v8::Local<v8::Value> OfficeClient::LoadDocument(v8::Isolate* isolate,
                                                 const std::string& path) {
+  office_->setOptionalFeatures(
+      LibreOfficeKitOptionalFeatures::LOK_FEATURE_NO_TILED_ANNOTATIONS);
+
   lok::Document* doc = GetDocument(path);
 
   if (!doc) {
-    doc = office_->documentLoad(path.c_str(), "en-US");
+    doc = office_->documentLoad(path.c_str(), "Language=en-US");
```

Comment:
Nice catch, but try to keep unrelated changes to a minimum in case we have to revert things.

It looks like they changed it to support a bunch of options. I'm not sure if they're actually included until the upstream changes are pulled though:
https://github.com/CollaboraOnline/online/blob/749fed14e0099141e72ae4b250686dbd67335616/kit/Kit.cpp#L1377-L1417

Reply:
I'll create a separate PR for this